token/cli/src/main.rs

```diff
@@ -1961,9 +2026,45 @@ fn app<'a, 'b>(
                             "Enable the mint authority to freeze associated token accounts."
                         ),
                 )
+                .arg(
+                    Arg::with_name("interest_bearing")
+                        .long("interest-bearing")
+                        .value_names(&["ADDRESS", "RATE"])
+                        .help("Specify the interest rate authority (or NONE) and rate in basis points"),
+                )
```

Comment:
youre right that this is a tricky interface. i dont like the double parameter, it feels confusing and un-unixy. im thinking the most reasonable thing to do is make the flag `--interest-rate`, accept just the rate, and make the authority same as mint authority (as is done with `--enable-freeze` and `--enable-close` already). users can use `set-authority` to change this (it turns out `command_authorize` doesnt support all the new types yet, but ill open a separate issue, dont worry about this for your own pr)